metadata/packer/packer.go

```diff
@@ -123,6 +131,13 @@ func (p *Packer) Pack(pkg *ctipackage.Package, destination string) error {
 				}
 			}
 
+			// Suport custom file exclude function
+			if p.FileExcludeFunction != nil {
+				if err := p.FileExcludeFunction(fsPath, e); err != nil {
```

Comment:
How does it work?

Reply:
it reacts on errors archiver.SkipDir, archiver.SkipFile to skip whole directory or just a single file